<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticWidget</class>
 <widget class="QWidget" name="StatisticWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StatisticWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="criteriaLayout">
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="dtEditFrom"/>
     </item>
     <item row="1" column="1">
      <widget class="QDateTimeEdit" name="dtEditTo"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>From</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>To</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="contentLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
